src/olympia/versions/models.py

```diff
@@ -543,6 +543,19 @@ def unreviewed_files(self):
             models.Q(version__addon__status__in=amo.UNDER_REVIEW_STATUSES,
                      status__in=under_review_or_lite))
 
+    def activity_log(self, action__in_filter=None):
+        from olympia.devhub.models import ActivityLog  # Becoz circular import.
+        alog = ActivityLog.objects.for_version(self)
+        if filter:
```

Comment:
You mean `if action__in_filter` ? Since tests are passing a test is obviously missing :)